go.mod

```diff
@@ -153,6 +153,8 @@ require (
 	k8s.io/client-go v0.20.0
 )
 
+require github.com/cenkalti/backoff v2.2.1+incompatible // indirect
```

Comment:
nit: move inside the other require.

Reply:
I just noticed that. It was done by VS Code. Very odd.